### ISSUE TITLE
Select map button not displayed

### ISSUE THREAD

**Issue by pekka-28** [okviz/SynopticPanel#42]
After inserting a Synoptic Panel Visual into a Power BI report, the Select map button (toolbar?) does not show and no image can be selected.

Similarly, when opening the sample at https://visuals.azureedge.net/app-store/synopticPanelByOKViz.1.5.0.0.pbix, the data is visible, but no image is displayed.

I have tried with Visual versions 1.5.0 and 1.5.0.0.

**Comment by danieleperilli** [comment]
The sample file works well for me. 
Regarding your report: you have to bind some data before loading images.

**Comment by pekka-28** [comment]
Daniele

I understand about the binding. I did this with my dataset and when I visualise a Table with several columns, the expected values are shown. At first I thought this was the problem until I saw that same behaviour for the sample report on the Office store. This supported my suspicion that the changes in the names from Legend and Details to Category and Subcategory are all that is different.

My suspicion is some configuration on my machine, but I have no idea where to look to identify the reason for the problem. I have looked through the Typescript and see the imageData property, but don’t understand enough about the Visual interfaces to see where the button is displayed to be able to look for why they do not show. Do you have any troubleshooting documentation?

I can confirm that the Visual shows a Local maps and Gallery toolbar at app.powerbi.com. The errors I reported were on Power BI Desktop.

Regards
Pekka Pihlajasaari
--
pekka@data.co.za           Data Abstraction (Pty) Ltd         +27 11 484 9664

**Comment by danieleperilli** [comment]
Which version of Power BI?

**Comment by pekka-28** [comment]
Daniele

I have appended the location and version.

When running the web client I also saw a message to use the online editor. This was not present on the Desktop reports. It is almost like the scripting is disabled preventing anything from working. I have tried by running the application as an administrator without any visible difference.

Regards
Pekka Pihlajasaari
--
pekka@data.co.za           Data Abstraction (Pty) Ltd         +27 11 484 9664
--
"C:\Program Files\Microsoft Power BI Desktop\bin\PBIDesktop.exe"
2.48.4792.721 (PBIDesktop)
